transforms/code/malware/ray/Dockerfile

```diff
@@ -49,8 +54,10 @@ COPY test-data/ test-data/
 ENV PYTHONPATH /home/ray
 
 # Put these at the end since they seem to upset the docker cache.
-ARG BUILD_DATE
-ARG GIT_COMMIT
 LABEL build-date=$BUILD_DATE
```

Comment:
can you put these Label lines at the end of the Dockerfile

Reply:
Done.